.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+  - type: input
+    id: operating_system
+    attributes:
+      label: OS
+      description: Operating system and version (eg Ubuntu 20.04, OSX Catalina)
+      placeholder: linux
+
+  - type: architecutre
+    id: cpu_arch
+    attributes:
+      label: CPU Architecture
+      description: CPU Architecture (eg: x86, x86-64, arm, etc).
+      placeholder: linux
+
+  - type: input
+    id: redis_version
+    attributes:
+      label: Redis Server Version
+      description: What version of redis-server are you running (redis-server --version)
+      placeholder: latest
+
+  - type: input
+    id: redis_module_version
```

Comment:
Could be several modules - so user can copy-paste? Maybe add a 'name' attribute for it?

Reply:
So this redis-module-version bit was specific to this module. But yes, I could see a version(s) of the thing being reported against.